src/nvim/screen.c

```diff
@@ -5711,20 +5711,22 @@ next_search_hl_pos(
 
   shl->lnum = 0;
   for (i = posmatch->cur; i < MAXPOSMATCH; i++) {
-    if (posmatch->pos[i].lnum == 0) {
+    ll_pos_T *pos = &posmatch->pos[i];
```

Comment:
`s/ll_pos_T/llpos_T/`  I'm telling ya, copy/paste from vim-patch.sh prepared patches is where it's at.  ;)